q2_dada2/assets/run_dada_single.R

```diff
@@ -104,7 +106,7 @@ truncLen <- as.integer(args[[5]])
 trimLeft <- as.integer(args[[6]])
 maxEE <- as.numeric(args[[7]])
 truncQ <- as.integer(args[[8]])
-maxLen <- if (args[[9]]=='Inf') Inf else as.integer(args[[9]])
+maxLen <- as.numeric(args[[9]]) # Allows Inf
```

Comment:
Isn't the argument being passed in `'Inf'`, not `Inf` (that is, a string, versus the sentinel value)? Did something change about how R parses these things?

Reply:
Yes, the `as.numeric` coercion handles `Inf` the string, i.e. as.numeric('Inf') == Inf
I made that change just to have more idiomatic consistency in the variable coercions.
The effect is identical to before.